react/features/invite/actions.js

```diff
@@ -11,41 +10,57 @@ declare var $: Function;
 declare var APP: Object;
 declare var config: Object;
 
+const CONFERENCE_ID_ENDPOINT = config.conferenceMapperUrl;
+const DIAL_IN_NUMBERS_ENDPOINT = config.dialInNumbersUrl;
+const MUC_URL = config && config.hosts && config.hosts.muc;
+
 /**
  * Opens the Invite Dialog.
  *
  * @returns {Function}
  */
 export function openInviteDialog() {
     return openDialog(InviteDialog, {
-        conferenceUrl: encodeURI(APP.ConferenceUrl.getInviteUrl()),
-        dialInNumbersUrl: config.dialInNumbersUrl
+        conferenceUrl: encodeURI(APP.ConferenceUrl.getInviteUrl())
     });
 }
 
 /**
- * Sends an ajax request for dial-in numbers.
+ * Sends an ajax requests for dial-in numbers and conference id.
  *
- * @param {string} dialInNumbersUrl - The endpoint for retrieving json that
- * includes numbers for dialing in to a conference.
  * @returns {Function}
  */
-export function updateDialInNumbers(dialInNumbersUrl) {
-    return dispatch => {
-        dispatch({
-            type: UPDATE_DIAL_IN_NUMBERS_REQUEST
+export function updateDialInNumbers() {
+    return (dispatch, getState) => {
+
+        if (!CONFERENCE_ID_ENDPOINT || !DIAL_IN_NUMBERS_ENDPOINT || !MUC_URL) {
+            return;
```

Comment:
That's a really good idea. I had removed the error display in the dialog. I had left this error catching for future use but now would be a good time to at least log the error.